lib/hammer_cli_katello/filter.rb

```diff
@@ -53,6 +53,15 @@ class CreateCommand < HammerCLIKatello::CreateCommand
       success_message _("Filter created")
       failure_message _("Could not create the filter")
 
+      validate_options do
+        organization_options = [:option_organization_id, :option_organization_name, \
+                                :option_organization_label]
+
+        if option(:option_repository_names).exist? || option(:option_content_view_name).exist?
+          any(*organization_options).required
```

Comment:
I'm against it only because I think that the message that Hammer will give is much more clear. It lets the user know that they can specify organization-id, label or name.

I'm pretty sure the error that they get back from the API will not have as much information, and be more confusing. The message tailored here I believe gives a better use experience.